Day2/BoxFactory.py

```diff
@@ -0,0 +1,30 @@
+class BoxFactory:
+    
+    packaging_dimensions = None
+    finished_packaging = []
+    
+    def __init__(self, packaging_dimensions):
+        self.packaging_dimensions = packaging_dimensions
+        self.finished_packaging = []
+
+    def make_box(self, length, width, height):
+        side1 = length*width
+        side2 = width*height
+        side3 = height*length
+        return (side1, side2, side3)
```

Comment:
its not a box ITS A LIE the box is a LIE

uj class: box es en nem a sideokat mentenem el hanem a l/w/ht tobb dolgot tudsz bloluk szamolni kevesbbe limital 

ha tudod h kellenek a sideok akkor kiszamolhatod oket rogton a box ctorban v lehet fved ami kiszamolja es elmenti